Makefile

```diff
@@ -141,6 +153,73 @@ liccheck:
 copyright-check:
 	python scripts/check_copyright.py
 
+####################
+### Clean and init commands
+####################
+
+.PHONY: clean
+clean: clean-build clean-pyc clean-test clean-docs
+
+.PHONY: clean-build
+clean-build:
+	rm -fr build/
+	rm -fr dist/
+	rm -fr .eggs/
+	rm -fr pip-wheel-metadata
+	find . -name '*.egg-info' -exec rm -fr {} +
+	find . -name '*.egg' -exec rm -fr {} +
+
+.PHONY: clean-docs
+clean-docs:
+#   Update when docs PR is merged to avoid conflicts
+# 	rm -fr docs/build/
+
+# This can be replaced with rmcache
+.PHONY: clean-pyc
+clean-pyc:
+	find . -name '*.pyc' -exec rm -f {} +
+	find . -name '*.pyo' -exec rm -f {} +
+	find . -name '*~' -exec rm -f {} +
+	find . -name '__pycache__' -exec rm -fr {} +
+	find . -name '.DS_Store' -exec rm -fr {} +
+
+.PHONY: clean-test
+clean-test:
+	rm -fr .tox/
+	rm -f .coverage
+	find . -name ".coverage*" -not -name ".coveragerc" -exec rm -fr "{}" \;
+	# rm -fr coverage.xml
+	rm -fr coverage_report/
+	rm -fr .hypothesis
+	rm -fr .pytest_cache
+	rm -fr .mypy_cache/
+	find . -name 'log.txt' -exec rm -fr {} +
+	find . -name 'log.*.txt' -exec rm -fr {} +
+
+v := $(shell pip -V | grep virtualenvs)
+
+.PHONY: new-env
```

Comment:
change to `new_env`

Reply:
Ok. Just tried to keep similarity with other commands in this file.